packages/flutter_splash_tizen/bin/file_utils.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:io';
+import 'package:yaml/yaml.dart';
+import 'package:xml/xml.dart';
+
+Map? loadYamlFileSync(String path) {
+  File file = File(path);
+  if (file.existsSync() == true) {
+    return loadYaml(file.readAsStringSync());
+  }
+  return null;
+}
+
+XmlDocument loadXMLFileSync(String path) {
+  File file = File(path);
+  if (file.existsSync() == true) {
+    return XmlDocument.parse(file.readAsStringSync());
+  }
+  throw FormatException("could not read $path");
```

Comment:
I think returning `null` is better. 
```suggestion
  return null;
```